tests/classification/test_accuracy.py

```diff
@@ -200,3 +200,89 @@ def test_wrong_params(top_k, threshold):
 
     with pytest.raises(ValueError):
         accuracy(preds, target, threshold=threshold, top_k=top_k)
+
+
+@pytest.mark.parametrize(
+    "preds_mc, target_mc, preds_ml, target_ml",
+    [
+        (
+            tensor([0, 1, 1, 1]),
+            tensor([2, 2, 1, 1]),
+            tensor([[1, 0, 1, 1], [0, 0, 1, 0]]),
+            tensor([[0.8, 0.2, 0.8, 0.7], [0.6, 0.4, 0.6, 0.5]])
+        )
+    ],
+)
+def test_different_modes(preds_mc, target_mc, preds_ml, target_ml):
+    acc = Accuracy()
+    acc(preds_mc, target_mc)
+    with pytest.raises(ValueError):
```

Comment:
```suggestion
    with pytest.raises(ValueError, match="The `average` has to be one of"):
```
pls add some part of the error message so we are clear that it the one Error